pkg/workflows/workflowconfig.go

```diff
@@ -1,39 +1,64 @@
 package workflows
 
 type WorkflowConfig struct {
-	AcrName           string
```

Comment:
why are we renaming this to Acr ? I think we can keep this as is

Reply:
I renamed it so that it more closely matches the variable name provided in the yaml templates. I'll change it back though, AcrName does sound better.